### PR TITLE
Openvino2020.1

### PR DESCRIPTION
**Pull request by priyanka-bagade**
updated the object detection cpp tutorial. Please validate and merge to master.

### CODE REVIEW

**Comment by avladim** [comment]
Hi Priyanka, I found a number of functional and cosmetic issues with this request:

1) The kernel set in the notebook is "Python 3", which cannot load qarpo. You should set the kernel to "Python 3 (Ubuntu)". In fact, we should find a way to prevent Jupyter from showing the "Python 3" kernel.

2) The notebook had output of executed cells — please clean output before saving the notebook and committing.

3) You added the script committed "object_detection_job.sh", but it is not needed because it should be produced by a cell beginning with "%writefile"

4) The path to the produced executable is "./build/intel64/object-detection-cpp", but the notebook has "./object-detection-cpp" — need to fix one or the other, otherwise the job fails

5) After I fixed the path (locally), the executable failed because "-fr" is not a recognized command line flag.

6) I removed "-fr" (locally), but the job failed because "-o" is not a recognized command line flag

At this point, I stopped trying to fix it and decided to send it back to you to fix the issues or let me know if I am doing something wrong with evaluation. 

Bottom line, I was not able to run the demo, neither out-of-the-box, nor after minor fixes.
